doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, Make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, Follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, Create an empty project folder called `projects`, navigate into it:
+
+Linux and Mac:
+
+```txt
+$ mkdir ~/projects
+$ cd ~/projects
+```
+Windows CMD:
+```txt
+> mkdir %USERPROFILE%\projects
+> cd %USERPROFILE%\projects
+```
+
+Windows PowerShell:
+
+```txt
+> mkdir $env:USERPROFILE\projects
+> cd $env:USERPROFILE\projects
+```
+
+Next, create a new source file in the `projects` folder and call it `hello_world.js`.
+
+If you’re using more than one word in your filename, use an underscore(`_`) to separate them for simplicity and avoid using the space character in file names. 
```

Comment:
Maybe "use an underscore (`_`) or a hyphen (`-`)"? Some style guides and practices (including Node.js repository) prefer hyphens, so we can mention both here.